rackcity/views/asset_views.py

```diff
@@ -137,8 +143,24 @@ def asset_detail(request, id):
     return JsonResponse(serializer.data, status=HTTPStatus.OK)
 
 
+def get_change_plan(change_plan_id):
+    change_plan = ChangePlan.objects.get(
+        id=change_plan_id
+        )
+    if not change_plan:
+        return JsonResponse(
+            {
+                "failure_message":
+                    Status.CREATE_ERROR.value + GenericFailure.INTERNAL.value,
+                "errors": "Invalid change_plan Parameter"
+            },
+            status=HTTPStatus.BAD_REQUEST
+        )
+    return change_plan
```

Comment:
I know python doesn't have typing so this is technically fine, but I think we should avoid returning inconsistent types...instead I would return a tuple with (change_plan, failure_response) and either return (None, failure_response) or (change_plan, None), then check if failure_response is None when this method is called to make sure you're catching the error. I've done this in other places in the code if you want an example